File: lab4/lab4/Matrix.cpp
#include "stdafx.h"
#include "Matrix.h"
#include <iostream>
using namespace std;


void Matrix::Generate()
{
	if (Value == nullptr) return;
	for (int i = 0; i < N; i++)
	{
		for (int j = 0; j < N; j++)
		{
			Value[i][j] = 1;
		}
	}
}

Matrix::Matrix(int count)
{
	N = count;
	Value = new int *[N];
	for (int i = 0; i < N; i++) {
		Value[i] = new int[N];
	}
	Generate();
}

Matrix::~Matrix()
{
}

int Matrix::getN()
{
	return N;
}

void Matrix::Output()
{
	if (Value == nullptr) return;
	for (int i = 0; i < N; i++)
	{
		for (int j = 0; j < N; j++)
		{
			cout << Value[i][j] << "\t";
		}
		cout << endl;
	}
}

File: lab09/lab9/Matrix.cs
using System;

namespace lab9
{
    class Matrix
    {
        private int n = 0;
        private int[,] value = null;
        //public int[,] Value { get => value; set => this.value = value; }
        //public int N { get => n; set => n = value; }

        public int getN()
        {
            return this.n;
        }

        public void setN(int n)
        {
            this.n = n;
        }

        public int[,] getValue()
        {
            return this.value;
        }

        public int getValue(int i, int j)
        {
            return this.value[i,j];
        }


        public void setValue(int[,] value)
        {
            this.value = value;
        }

        public void setValue(int i, int j, int value)
        {
            this.value[i,j] = value;
        }
        

        public Matrix(int n)
        {
            this.n = n;
            value = new int[n, n];
            //this.Generate();
        }
        

        public void Generate()
        {
            if (value == null) return;
            for (int i = 0; i < n; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    value[i, j] = 1;
                }
            }
        }

        public void Output()
        {
            if (value == null) return;
            for (int i = 0; i < n; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    Console.Write(value[i, j].ToString() + "\t");
                }
                Console.WriteLine();
            }

        }

    }
}